friendsofsymfony/user-bundle/2.0/config/packages/fos_user.yaml

```diff
@@ -0,0 +1,12 @@
+# Read the documentation: https://symfony.com/doc/master/bundles/FOSUserBundle/index.html
+fos_user:
+    db_driver: orm # other valid values are 'mongodb' and 'couchdb'
+    firewall_name: main
+    user_class: AppBundle\Entity\User
+    from_email:
+        address: "%env(MAILER_USER)%"
+        sender_name: "%env(MAILER_USER)%"
+
+services:
+    validator.builder:
```

Comment:
That's absolutely not how the recipe should work. Instead, the package itself should require the Symfony package that installs this service, which is `symfony/validator` in this case.

This is a problem from the FOSUserBundle side that should be fixed before a recipe can be created.

Reply:
uh... I understand, sorry.

Reply:
This service should be removed. You use add `symfony/validator` to the dependencies section in your composer.json.